lib/mix/lib/mix/tasks/new.ex

```diff
@@ -261,33 +256,28 @@ defmodule Mix.Tasks.New do
     use Mix.Project
 
     def project do
-      [app: :<%= @app %>,
-       version: "0.1.0",
-       elixir: "~> <%= @version %>",
-       build_embedded: Mix.env == :prod,
-       start_permanent: Mix.env == :prod,
-       deps: deps()]
+      [
+        app: :<%= @app %>,
+        version: "0.1.0",
+        elixir: "~> <%= @version %>",
+        start_permanent: Mix.env == :prod,
+        deps: deps()
+      ]
     end
 
-    # Configuration for the OTP application
-    #
-    # Type "mix help compile.app" for more information
+    # Run "mix help compile.app" to learn about applications.
     def application do
-      # Specify extra applications you'll use from Erlang/Elixir
-  <%= @otp_app %>
+      [
+        extra_applications: [:logger]<%= @sup_app %>
+      ]
     end
 
-    # Dependencies can be Hex packages:
-    #
-    #   {:my_dep, "~> 0.3.0"}
-    #
-    # Or git/path repositories:
-    #
-    #   {:my_dep, git: "https://github.com/elixir-lang/my_dep.git", tag: "0.1.0"}
-    #
-    # Type "mix help deps" for more examples and options
+    # Run "mix help deps" to learn about dependencies.
     defp deps do
-      []
+      [
+        # {:dep_example, "~> 0.3.0"},
```

Comment:
Should we call these something like `dep_from_hex` and `dep_from_git`?

Or we could use a real dependency such as `ex_doc` .